src/options.h

```diff
@@ -109,6 +109,7 @@ struct options
   char **domains;               /* See host.c */
   char **exclude_domains;
   bool dns_cache;               /* whether we cache DNS lookups. */
+  char resolve_host;            /* manual host resolution */
```

Comment:
`char** resolve_host`
also you need to allocate memory for this field somewhere

Reply:
I'll remove this as there's no need to store anything, idea is to parse input and immediately store a pair `(host, ip)` in a dns cache.